### PR TITLE
Bug/package fix

### PR DESCRIPTION
**Pull request by ashtilawat**
## What work was done? 
- Package.json fix!
- Code removal for index.js required for single player mode. 

Update version to the file required for auth0.

### CODE REVIEW

**Review by kendra4227 — DISMISSED** [review]
Good update on the name and version

**Comment by tinomen** [comment]
Looks like more than the package.json was changed ;)

**Review by kendra4227 — COMMENTED** [review]
Will you be adding anything inside the row?

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "story-squad",
```
Comment (ghost) [inline package.json:2]: Another "story squad" sighting bites the dust!

**Review by ghost — DISMISSED** [review]

**Review by artofmayhem — DISMISSED** [review]

File: src/index.js [hunk 2]
```diff
@@ -113,8 +111,6 @@ function App() {
         <Route exact path="/gamemode" component={Gamemode} />
 
         <Route path="/gameification" component={GameificationMain} />
```
Comment (ethankarp) [inline src/index.js]: We're not using "gameification" anymore right?

Comment (ashtilawat) [reply src/index.js]: Just resolved this when resolving merge conflicts.

Comment (ashtilawat) [reply src/index.js]: Good callout and attention to detail.

**Review by artofmayhem — APPROVED** [review]
this looks good thank you so much

**Review by rhiannonstanford — APPROVED** [review]
It looks good.  Thank you for the updated fix.

File: src/index.js [hunk 3]
```diff
@@ -42,7 +42,6 @@ import { NotFoundPage } from './components/pages/NotFound';
 import { ParentFaq } from './components/pages/ParentFaq';
 import { ParentContact } from './components/pages/ParentContact';
 import { NewParentDashboard } from './components/pages/NewParentDashboard';
-import { ParentDashFaq } from './components/pages/ParentDashFaq';
```
Comment (ethankarp) [inline src/index.js:45]: I'm afraid the app is breaking because the route that attempts to render this component is still in the index.js file on line 207

**Review by ethankarp — CHANGES_REQUESTED** [review]

**Review by Yonathan-Admasu728 — COMMENTED** [review]
Great fix in all of the 3 files. Thank you

**Review by Yonathan-Admasu728 — COMMENTED** [review]
Great fix in all of the 3 files. Thank you

**Pull request merged**